### PR TITLE
Fix topic title

### LINKED ISSUE #2646: Agenda: detail view don't include numbering in title
**Description:** System generated numbers for topics are not included in the topic titles in the detail view. This is wrong. The title should include the generated numbers (with prefix).

**Reproduction:**
1. open a meeting
2. open agenda
3. create a topic
4. head back to agenda > open the three-dot-menu > choose numbering > confirm dialog > topic should have a number
5. open the topic > title doesn't include number 

**What should happen:**
If the user generates automatic numbers für topics, than they should appear in the topic title in the topic detail view. 

**How should it look like - see comparison to OS3:**
Screenshot_20230816_120015

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2646 

@MSoeb should the number be included only in that title or also everywhere else where the topic title is displayed?

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/agenda/agenda.subscription.ts [hunk 1]
```diff
@@ -108,7 +108,8 @@ export const getTopicDetailSubscriptionConfig: SubscriptionConfigGenerator = (..
             {
                 idField: `list_of_speakers_id`,
                 ...listOfSpeakersSpeakerCountSubscription
-            }
+            },
+            `agenda_item_id`
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/agenda/agenda.subscription.ts:112]:
```suggestion
            { idField: `agenda_item_id`, fields: [`item_number`] }
```

**Review by bastianjoel — CHANGES_REQUESTED** [review]

**Review by bastianjoel — APPROVED** [review]

**Comment by Elblinator** [comment]
The number should be included everywhere 

Can you indicate in a comment or a commit where the numbering would be in included?

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]

File: client/src/app/site/pages/meetings/view-models/base-projectable-model.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export abstract class BaseProjectableViewModel<M extends BaseModel = any> extend
      * @returns the projector title used for managing projector elements.
      */
     public getProjectorTitle(projection: Projection): ProjectorTitle {
+        console.log(`PROJECTOR TITLE!!!`, this);
```
Comment (rrenkert) [inline client/src/app/site/pages/meetings/view-models/base-projectable-model.ts:29]: Cleanup

**Review by rrenkert — DISMISSED** [review]
Number and title are now visible in projector and PDF export so works as expected.

**Pull request merged**